test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -275,6 +275,17 @@ def test_successful_purchase_with_tokenized_visa
     assert_not_nil(purchase.params.dig('charges', 'data')[0]['payment_method_details']['card']['network_token'])
   end
 
+  def test_successful_purchase_with_network_token_cc
```

Comment:
Can you also unit test were we confirm that we pass last_4 even when last_4 is not present in options? Similar to this [test](https://github.com/activemerchant/active_merchant/blob/3b28d8622f26121b0eff29d4f0b6bc9be6c817cd/test/unit/gateways/stripe_payment_intents_test.rb#L530)